src/python/setup.cfg

```diff
@@ -0,0 +1,17 @@
+[metadata]
+description-file = README.md
+
+[flake8]
+max-line-length = 79
+
+[mypy]
+python_version = 3.7
```

Comment:
Sorry, missed in previous reviews :-(
Shouldn't there be a 3.9 version?

Reply:
you are right, more comments is better :) Improve the PR.